README.md

```diff
@@ -22,4 +22,5 @@ Show your support
 Give a ⭐️ if you like this project!
 
 Acknowledgment:
-More thanks for the knowledge on styling by using Figma
\ No newline at end of file
+More thanks for the knowledge on styling by using Figma
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you update your README with information regarding the project you are building now. You can use this [Microverse template](https://github.com/microverseinc/readme-template)